testsuite/tests/kuadrant/authorino/operator/clusterwide/test_all_namespace_api_key.py

```diff
@@ -7,6 +7,8 @@
 
 from testsuite.httpx.auth import HeaderApiKeyAuth
 
+pytestmark = [pytest.mark.authorino, pytest.mark.standalone]
```

Comment:
I'd better explicitly specify which test can run on kuadrant with kuadrant mark instead of specifying which test can't run with it. So every test will contain every mark that this test is able to run with (e.g. limitador will have kuadrant and limitador marks, authorino tests will have kuadrant and authorino marks, but authorino metrics tests will have only authorno mark). Like a tree structure of marks.

Reply:
Tests that run on the Kuadrant deployment are much more prevalent than those who do not